digitalocean/datasource_digitalocean_container_registry.go

```diff
@@ -0,0 +1,40 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func dataSourceDigitalOceanContainerRegistry() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceDigitalOceanContainerRegistryRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
It might be useful to also export a computed value for the registry's endpoint (e.g. `registry.digitalocean.com/foo`). It would match doctl's behaviour. 

https://github.com/digitalocean/doctl/blob/703436d0e4ad535eb31b00bd34c0e599eec9a335/do/registry.go#L31

Reply:
Maybe also expose godo's `DockerConfigJSON` for the registry `? That would allow using the Docker config in a provisioner or another resource.